wgsl/index.bs

```diff
@@ -489,8 +489,11 @@ The following is a list of keywords which are reserved for future expansion.
   <tr><td>`NOT_EQUAL`<td>!==
   <tr><td>`GREATER_THAN`<td>>
   <tr><td>`GREATER_THAN_EQUAL`<td>>=
+  <tr><td>`GREATER_GREATER`<td>>>
```

Comment:
Would be nice to make these names consistent. I.e. why `ARROW` and not `MINUS_GREATER`? Similarly, there is `ATTR_LEFT` and not `BRACKET_LEFT_LEFT`.

If `GREATER_GREATER` is in fact the desired level at which the naming is done, then this would be something to follow-up. I wanted to raise it here in case `GREATER_GREATER` should be replaced by something like `ARITHMETIC_SHIFT_RIGHT`.

Reply:
Sure, a more semantic name would be a better choice.

Reply:
Fixed in https://github.com/gpuweb/gpuweb/pull/792/commits/a1f03bdff1ddebe53fb1699bc08dabd206b13028